plugins/modules/zos_job_submit.py

```diff
@@ -124,7 +124,7 @@
 RETURN = r"""
 jobs:
   description:
-     List of jobs output.
+     List of jobs output, or a single NOTFOUND record.
```

Comment:
Maybe something like this:
description:
List of the jobs output or a single NOTFOUND record when the job has not been run.

NOTFOUND could change depending on my earlier comments

Reply:
@richp405 - seems that this one has been missed yet the other one is fixed or you are waiting for approval before updating others similar comments
https://github.com/ansible-collections/ibm_zos_core/pull/260#discussion_r629592125

Reply:
I forgot about submit.